exchanges/order/orders.go

```diff
@@ -740,6 +718,33 @@ func (t Type) String() string {
 	}
 }
 
+// String implements the stringer interface.
+func (t TimeInForce) String() string {
```

Comment:
Because this can be flagged with Post only and GTT, GTD, GTC,  it needs to be able to be a delimitated string and umarshal multiple options. Please add a test.